packaging/suse/trento.spec

```diff
@@ -0,0 +1,106 @@
+#
+# spec file for package trento
+#
+# Copyright (c) 2021 SUSE LLC
+#
+# All modifications and additions to the file contributed by third parties
+# remain the property of their copyright owners, unless otherwise agreed
+# upon. The license for this file, and modifications and additions to the
+# file, is the same license as for the pristine package itself (unless the
+# license for the pristine package is not an Open Source License, in which
+# case the license is the MIT License). An "Open Source License" is a
+# license that conforms to the Open Source Definition (Version 1.9)
+# published by the Open Source Initiative   .
+
+# Please submit bugfixes or comments via https://bugs.opensuse.org/
+#
+
+
+Name:           trento
+# Version will be processed via set_version source service
+Version:        0
+Release:        0
+License:        Apache-2.0
+Summary:        An open cloud-native web console improving on the life of SAP Applications administrators .
+Group:          System/Monitoring
+URL:            https://github.com/trento-project/trento
+Source:         %{name}-%{version}.tar.gz
+Source1:        vendor.tar.gz
```

Comment:
Where are these source files coming from?
Are they created during the CI process?
If they are pushed manually to the repo, I would at least put a comment in top of them

Reply:
They are created by the obs services, when calling `$ osc service dr` command. We run this command in the CI, but it could be run manually the same.

Reply:
Nice!